### ISSUE TITLE
(bug) background-image with double quotes

### ISSUE THREAD

**Issue by mifi** [website-scraper/node-website-scraper#403]
input:
```html
<div style="width: 300px; height: 300px; background-image:url(&quot;https://via.placeholder.com/350x150&quot;)" class="image-background bg-image-1"></div>
```

output:
```html
<div style="width: 300px; height: 300px; background-image:url("350x150")" class="image-background bg-image-1"></div>
```
it doesn't escape the double quotes, producing invalid HTML

**Comment by s0ph1e** [comment]
Hi @mifi 

Thank you for reporting this issue 👍 

Looks like it may be caused by bug in cheerio https://github.com/cheeriojs/cheerio/issues/957
but can be fixed on website-scraper side in this place https://github.com/website-scraper/node-website-scraper/blob/1155dde11088f0dbd2cc68479bc01e030b68d28d/lib/resource-handler/html/html-source-element.js#L40-L42

I'll take closer look on it later. Also contributions are welcome :)

**Comment by s0ph1e** [comment]
Hi @mifi 👋 

Issue with double quotes is fixed in version [4.2.1](https://github.com/website-scraper/node-website-scraper/releases/tag/v4.2.1)

**Comment by mifi** [comment]
awesome, thanks
